transform/transformers/pck_transformer.py

```diff
@@ -171,6 +175,19 @@ def preprocess_comments(self):
         self.data = data
         return self.data
 
+    def calculate_total_playback(self):
+        """
+        Calculates the total value of acquisitions and proceeds from disposals
+        from submitted data for QCAS.
+        :return:
+        """
+        if self.survey.get('survey_id') == self.qcas_survey_id:
+            total_acquisitions = sum(Decimal(value) for q_code, value in self.data.items() if q_code in self.qcas_acquisitions_questions)
+            total_disposals = sum(Decimal(value) for q_code, value in self.data.items() if q_code in self.qcas_disposals_questions)
```

Comment:
Can values ever be None?

Reply:
For this scenario as the questions are mandatory, answers must be `>=0`.